query/src/test/scala/hmda/query/model/institutions/InstitutionQueryGenerators.scala

```diff
@@ -7,20 +7,60 @@ object InstitutionQueryGenerators {
   implicit def institutionQueryGen: Gen[InstitutionQuery] = {
```

Comment:
I'm not sure about this one, but I assume we will be testing this in the future.